external/signal/resample.m

```diff
@@ -119,7 +119,7 @@
 % pre and postpad filter response
 
 nz_pre = floor(q-mod(L,q));
-hpad = [zeros(nz_pre,1);h];
+hpad = [zeros(nz_pre,1);h'];
```

Comment:
The vector `h` should not be transposed, but should be forced to always be a column vector. That way it works both when people pass h as a row and as a column, in line with the Mathworks implementation.